doc/nrf/ug_app_dev.rst

```diff
@@ -32,23 +32,13 @@ See the following links for information about the different building blocks ment
 ===============
 
 The |NCS| adds some functionality on top of the Zephyr build and configuration system.
-You must be aware of these additions when you start writing your own applications based on this SDK.
-
-  * The |NCS| provides an additional :file:`boilerplate.cmake` that must be included before Zephyr's in the :file:`CMakeLists.txt` file of your application::
+Those additions are automatically included into the Zephyr build system using a :ref:`zephyr:cmake_extension_package`.
```

Comment:
I see that :ref:`zephyr:cmake_extension_package` is not linking. Not sure if we should be linking to https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/extensions.cmake - In that case, we can replace the existing text with the following suggested text and add the following entry in the nrf\doc\links.txt file under **Links to Zephyr** section:
.. _`CMake extension package`: https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/extensions.cmake
```suggestion
These additions are automatically included into the Zephyr build system using the `CMake extension package`_.
```

Reply:
It links correctly when I build the docs locally.
Did you remember to build EVERYTHING according to the guide, and with no errors on missing tools (remember requirements.txt) ?
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_build.html#building-documentation-output

Reply:
Yes.. Looks like there was a problem which was fixed by a west update and a clean build.